docs/topics/templates/document-attributes.adoc

```diff
@@ -72,22 +72,21 @@ endif::[]
 :ocp-short: OpenShift Container Platform
 :ocp-first: {ocp-full} (RHOCP)
 
-// This book is on GitHub wiki only
-:CoreDevelopmentBookName: Core Development Guide
-
 // ********************
 // * Product versions *
 // ********************
 // Commenting out as these attributes are not currently used:
 // :ProductRelease: 4
 // :ProductVersion: 4.0
 //Questions
-:ProductDistributionVersion: 5.3.0.Final
-:ProductDistribution: mta-cli-{ProductDistributionVersion}
+:ProductDistributionVersion: 6.0.0.GA-redhat-00002
```

Comment:
You need to place these values in the ifdef statements for MTA and MTR
You can add these MavenProductValuyes: MTA = 6.0.0.GA-redhat-00002 and MTR = 1.0.0.GA-redhat-00001